### DISCUSSION TITLE
multi-vendor/multi-tenancy application using filament

### DISCUSSION THREAD

**Discussion by sajjad201** [filamentphp/filament#6997] [General]
Hi 
I just get to know about filament it seems good to jump quickly on building the websites in laravel with this packge. 
I just go through basic functionalities of it but before adopting it I have few questions. 

I have to build a hostel management system using filament which will be multi-vendors/multi-tenancy which means there will be multiple hostel owners who will be managing their hostels, bookings, customers, reports etc.

Super admin will be able to manage roles, permissions, approve hostels/hostel owners and should be able switch their account into hostel owner account.  
So what are the pre-requisites to builds this website with apis for mobile app in flutter using the filament and with some other packages/plugins. 

Can you please explain is it possible to achieve it ? will it provide ease in building apis for mobile app as well?

**Comment by sawirricardo** [comment]
it's possible. simple add a "team" model (or hostel, in your case). This will be another topic, but the big idea is like that. Mobile app with flutter simple just expose the laravel's api to your flutter app, so the answer is possible.
